lib/resources/r4/related_person.yaml

```diff
@@ -65,11 +62,13 @@ fields:
     description: The nature of the relationship between a patient and the related person. 
     terminology:
     - display: PatientRelationshipType
-      system: http://hl7.org/fhir/relatedperson-contactrelationship
+      system:
+        - http://terminology.hl7.org/CodeSystem/v2-0131
+        - http://terminology.hl7.org/CodeSystem/v3-RoleCode
       info_link: http://hl7.org/fhir/r4/valueset-relatedperson-relationshiptype.html
     - display: Millennium Person Relationship Types
       system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/351 
       info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes/#code-set-351-person-relationship-types
     - display: Millennium Person Relationship Types
```

Comment:
Millennium Person Relationships

Reply:
This is also currently consistent with how it is shown on the Patient page. Do you still want to change it?

Reply:
it makes sense to label this URL as Millennium Person Relationship Types
https://fhir.cerner.com/millennium/r4/proprietary-codes/#code-set-351-person-relationship-types

but it does not make sense to label this URL as Millennium Person Relationship Types
https://fhir.cerner.com/millennium/r4/proprietary-codes/#code-set-40-person-relationships

it would be good to change this in Patient as part of this PR

Reply:
Second Max

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/pull/284/commits/42aef5d4a9d465f7bcd078c00429693679834587)

Reply:
> it would be good to change this in Patient as part of this PR

seems unneeded as the Patient display is 'Millennium Person Relationship Types' for a URL of https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-351-person-relationship-types. unclear what this comment refers to https://github.com/cerner/fhir.cerner.com/pull/284#discussion_r322408013